src/elm/Routes/LandingPage.elm

```diff
@@ -8,13 +8,31 @@ import Types exposing (..)
 
 landingPage : Model -> Html Msg
 landingPage model =
-    div [ class "hands" ]
-        [ div [ class "purple-background w-100 h-100 " ]
-            [ div [ class "white b f2 tc pl3 pr3 pt5 tracked" ]
-                [ text "We are here to help." ]
-            , div [ class "white-line pb4" ] []
-            , whiteButton "SEE REAL STORIES" "quotes"
-            , whiteButton "SEE OUR SERVICES" "services"
-            , whiteButton "CONTACT US" "talk-to-us"
+    div []
+        [ div [ class "hands dn-l" ]
+            [ div [ class "purple-background w-100 h-100 " ]
+                [ div [ class "white b f2 tc pl3 pr3 pt5 tracked" ]
+                    [ text "We are here to help." ]
+                , div [ class "white-line pb4" ] []
+                , whiteButton "SEE REAL STORIES" "quotes"
+                , whiteButton "SEE OUR SERVICES" "services"
+                , whiteButton "CONTACT US" "talk-to-us"
+                ]
+            ]
+        , div [ class " group dib-l dn" ]
```

Comment:
Can probably make this a bit less repetitive, let's go through it in the morning